src/RoleObjects/AccessibilityObject.js

```diff
@@ -14,21 +15,25 @@ export default class AccessibilityObject {
     this._areKeyEventsEnabled = false;
 
     /**
-     * Fields with relatively fixed values that should go into the React props for the element translation of this object.  This is done as an object for easy merging with the rest of the props
+     * Fields with relatively fixed values that should go into the React props for
+        the element translation of this object.  This is done as an object for easy
```

Comment:
Nit: each line of a function comment should start with `*` and they should be aligned with the first `*` of the opening of the function comment